docs/sources/tutorials/alerting-get-started/alerting-get-started.md

```diff
@@ -0,0 +1,144 @@
+
+
+1. Clone the github.com/grafana/tutorial-environment repository.
+
+    ```
+    git clone https://github.com/grafana/tutorial-environment.git
+    ```
+
+1. Change to the directory where you cloned this repository:
+
+    ```
+    cd tutorial-environment
+    ```
+
+1. Make sure Docker is running:
+
+    ```
+    docker --version
+    ```
+
+    This command will display the installed Docker version if the installation was successful. 
+1. Start the sample application:
+
+    ```
+    docker compose up -d
+    ```
+
+    The first time you run docker compose up -d, Docker downloads all the necessary resources for the tutorial. This might take a few minutes, depending on your internet connection.
+    Note: If you already have Grafana, Loki, or Prometheus running on your system, then you might see errors because the Docker image is trying to use ports that your local installations are already using. Stop the services, then run the command again.
+
+1. Ensure all services are up-and-running:
+
+    ```
+    docker compose ps
+    ```
+
+    In the State column, it should say Up for all services.
+
+1. Finally, the Grafana News app should be live on localhost:8081.
+
+## Generate some data
+
+### Grafana News
+
+Grafana News is a straightforward application created to demonstrate the observation of data using the Grafana stack. It achieves this by generating web traffic through activities such as posting links and voting for your preferred ones
+
+To add a link:
+
+1. In Title, enter Example.
+1. In the URL, enter https://example.com.
+1. Click Submit to add the link. The link will appear listed under the Grafana News heading.
+To vote for a link, click the triangle icon next to the name of the link.
+
+### Log in to Grafana
+
+Besides being an open-source observability tool, Grafana has its own built-in alerting service.   This means that you can receive notifications whenever there is an event of interest in your data, and even see these events graphed in your visualizations.
+
+1. In your browser, simply navigate to localhost:3000. You should get logged in automatically
+
+### Create the Alert
+
+Before we get started, it’s important to note that Grafana Alerting supports two alert rule types: Grafana-managed alert rules, which allow you to create alerts that can act on data from multiple data sources, and data source-managed alert rules, which are alert rules created and stored within the data source itself. In our case, our data source is Prometheus, so we will be creating a Grafana-managed alert rule. This is because Grafana uses Prometheus Alertmanager as the default Alertmanager, which handles alerts sent by client applications, such as the Prometheus server. Moreover, the exported metrics can be queried using the Prometheus Query language (PromQL)  
+
+1. In Grafana, toggle the menu at the top left side of the screen, and navigate to Alerting > Alert rules. Click on  + New alert rule.
+
+#### Enter alert rule name
```

Comment:
under each section i'd perhaps add an intro sentence about what the goal is in each. why do you enter a name? what is the alert condition for? etc